apps/api/src/app/layouts/usecases/update-layout/update-layout.use-case.ts

```diff
@@ -24,6 +25,10 @@ export class UpdateLayoutUseCase {
     const databaseEntity = await this.getLayoutUseCase.execute(getLayoutCommand);
 
     const patchedEntity = this.applyUpdatesToEntity(this.mapToEntity(databaseEntity), command);
+    const hasBody = patchedEntity.content.includes('{{{body}}}');
+    if (!hasBody) {
+      throw new ApiException('Layout content must contain {{{body}}}');
+    }
```

Comment:
It would be better if we could validate this earlier so we can avoid to execute the database calls. It can go like this for now. We can improve performance later.